easybuild/tools/build_log.py

```diff
@@ -160,6 +180,15 @@ def init_logging(logfile, logtostdout=False, testing=False):
     return log, logfile
 
 
+def get_eb_logger(name=None, *args, **kwargs):
```

Comment:
why introduce this function? if you are going to change all source files and replace calls to `fancylogger.getLogger(name) with calls to `get_eb_logger(name)`, then you can just replace them with `fancylogger.getLogger('easybuild.' + name)` and spare yourself adding another function...